.storybook/preview.js

```diff
@@ -10,7 +10,12 @@ const preview = {
 				color: /(background|color)$/i,
 				date: /Date$/i
 			}
-		}
+		},
+    options: {
+      storySort: {
+        order: ['Introduction', 'Theming/Theme', 'Theming', 'Components', 'Layout', 'Utils', 'Actions', 'Stores'],
```

Comment:
the first two items (intro, theme) felt like they should be first because you need to understand them before you can use anything else, and from there i just went generally in terms of things that felt like they were core/main things -> more supportive stuff. but i didn't do much thinking about the order of 'Utils, Actions, Stores' and we could really arrange them any which way.